src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -885,8 +910,6 @@
     @Deprecated
     public static final Key<Value<SkeletonType>> SKELETON_TYPE = KeyFactory.fake("SKELETON_TYPE");
 
-    public static final Key<Value<UUID>> SKIN_UNIQUE_ID = KeyFactory.fake("SKIN_UNIQUE_ID");
```

Comment:
Is it worth deprecating this rather than just removing it?

Reply:
7 months on, how is this looking?